test/gtest/ucp/ucp_test.cc

```diff
@@ -843,6 +844,11 @@ ucs_status_t ucp_test_base::entity::listen(listen_cb_type_t cb_type,
         params.conn_handler.cb    = reject_conn_cb;
         params.conn_handler.arg   = reinterpret_cast<void*>(this);
         break;
+    case LISTEN_CB_CUSTOM:
+        params.field_mask        |= UCP_LISTENER_PARAM_FIELD_CONN_HANDLER;
+        params.conn_handler.cb    = custom_cb->cb;
+        params.conn_handler.arg   = custom_cb->arg;
```

Comment:
seems cb_type is no longer needed by this function.
the callers of this function should always just pass cb
(pass only cb, and arg is always `cast<void*>(this)`)

Reply:
There's still test for `accept_ep_cb` that is of different type than `conn_handler.cb`